core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -25,9 +25,9 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
   <j:invokeStatic var="ds" className="jenkins.model.DownloadSettings" method="get"/>
-  <form method="post" action="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}">
-    ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  <f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true">
     <f:submit value="${%Check now}" />
```

Comment:
An `<input type="submit">` inside an `<a href="…">`? I do not think so. Did you actually try this? The body of `<f:link>` is just supposed to be a label. The actual POSTing happens in `link.js` (`link.post`) via on `onclick` handler.